src/ngraph/frontend/onnx_import/op/xor.hpp

```diff
@@ -33,14 +34,19 @@ namespace ngraph
             {
                 inline NodeVector logical_xor(const Node& node)
                 {
-                    NodeVector ng_inputs{ngraph::op::numpy_style_broadcast(node.get_ng_inputs())};
-                    auto left = ng_inputs.at(0);
+                    auto left = node.get_ng_inputs().at(0);
                     auto not_left = std::make_shared<ngraph::op::Not>(left);
-                    auto right = ng_inputs.at(1);
+                    auto right = node.get_ng_inputs().at(1);
                     auto not_right = std::make_shared<ngraph::op::Not>(right);
                     return {std::make_shared<ngraph::op::Or>(
-                        std::make_shared<ngraph::op::And>(left, not_right),
-                        std::make_shared<ngraph::op::And>(not_left, right))};
+                        std::make_shared<ngraph::op::And>(
```

Comment:
ngraph have `xor` operator already, can we use it instead of this bunch of code?